# DP: arm/mach-ixp4xx portion of the beeper support for the NSLU2.

# Is in Linus' git, will be in 2.6.16

Beeper support for the NSLU2.

Signed-off-by: Alessandro Zummo <a.zummo@towertech.it>

 arch/arm/mach-ixp4xx/nslu2-setup.c |    7 +

--- linux-nslu2.orig/arch/arm/mach-ixp4xx/nslu2-setup.c	2006-01-23 02:13:55.000000000 +0100
+++ linux-nslu2/arch/arm/mach-ixp4xx/nslu2-setup.c	2006-01-25 11:32:14.000000000 +0100
@@ -52,6 +52,12 @@ static struct platform_device nslu2_i2c_
 	.num_resources		= 0,
 };
 
+static struct platform_device nslu2_beeper = {
+	.name			= "ixp4xx-beeper",
+	.id			= NSLU2_GPIO_BUZZ,
+	.num_resources		= 0,
+};
+
 static struct resource nslu2_uart_resources[] = {
 	{
 		.start		= IXP4XX_UART1_BASE_PHYS,
@@ -99,6 +105,7 @@ static struct platform_device *nslu2_dev
 	&nslu2_i2c_controller,
 	&nslu2_flash,
 	&nslu2_uart,
+	&nslu2_beeper,
 };
 
 static void nslu2_power_off(void)

